src/pages/ProjectSettings.tsx

```diff
@@ -110,10 +112,18 @@ export const ProjectSettings: React.FC = () => {
     ? projectType === ProjectType.Project || repoData
     : repoData;
 
+  const repoDisplay = `${repoData?.repoSettings?.projectRef?.owner}/${repoData?.repoSettings?.projectRef?.repo}`;
```

Comment:
Can we do this instead. It makes the final statement more readable.  
 ```suggestion
 const { repoSettings } = repoData || {};
 const { projectRef } = repoSettings || {};
 const { owner, repo } = projectRef || {}
  const repoDisplay = `${owner}/${repo}`;
```